static/app/components/searchSyntax/parser.tsx

```diff
@@ -389,12 +389,15 @@ export class TokenConverter {
     };
   };
 
-  tokenFreeText = (value: string, quoted: boolean) => ({
-    ...this.defaultTokenFields,
-    type: Token.FreeText as const,
-    value,
-    quoted,
-  });
+  tokenFreeText = (value: string, quoted: boolean) => {
+    return {
+      ...this.defaultTokenFields,
+      type: Token.FreeText as const,
+      value,
+      quoted,
+      invalid: this.checkInvalidFreeText(value),
+    };
+  };
```

Comment:
nit: I would leave the style as is, since that's how the rest of the file is
```suggestion
  tokenFreeText = (value: string, quoted: boolean) => ({
    ...this.defaultTokenFields,
    type: Token.FreeText as const,
    value,
    quoted,
    invalid: this.checkInvalidFreeText(value),
  });
```

Reply:
sure... I believe I was debugging with console.log and didn't revert afterward..sorry about that